ui/admin/config/environment.js

```diff
@@ -148,6 +148,7 @@ module.exports = function (environment) {
     ENV.featureFlags['ssh-target'] = true;
     ENV.featureFlags['vault-worker-filter'] = true;
     ENV.featureFlags['target-worker-filters-v2'] = true;
+    ENV.featureFlags['target-worker-filters-v2-ingress'] = true;
```

Comment:
NOT BLOCKING Question: Don't we need the flag turned on for test as well? I see the CI passed the test, I also see the test are passing in local but this flag, as per my understanding is turned off (`false`) within the test environment.

Reply:
We discussed controlling feature flags as close to the test cases as possible. You can see examples of it in this [PR](https://github.com/hashicorp/boundary-ui/pull/1585/files#diff-7c68e3f23829508db0431c1373bb7905a28b9d8bb60a42a245e4cd88b412743e). I use `hooks.beforeEach()` to set the feature flags for those test cases. I actually wasn't sure if I should leave this flag on for `development` or not. Happy to let the team decide which features we have when using vercel deployments.

Reply:
@cameronperera Amazing!! I even like more this approach. I remember we discuss about it, but it skip to me you add it here.

Would you mind to add a task on Jira or somewhere we will not forget to document this pattern? I am sure others we will need 😄

Reply:
I think an epic or story has been made but if not I will do so.

Reply:
https://hashicorp.atlassian.net/browse/ICU-7605